actionview/lib/action_view/helpers/asset_tag_helper.rb

```diff
@@ -268,6 +262,8 @@ def image_tag(source, options = {})
         tag("img", options)
       end
 
+      # This method will be deprecated in future versions of Rails.
```

Comment:
I meant that the comment says this method will eventually be deprecated, when it’s actually deprecated in this PR.

The deprecation warning says this method be removed in 5.2; it should say 6.0.